cmd/zap/cmd/footer.go

```diff
@@ -21,6 +21,7 @@ import (
 )
 
 // footerCmd represents the footer command
+// - support the sectionsIndexOffset over here?
```

Comment:
since the fieldsIndexOffset and sectionsIndexOffset both point to the same info, I think its better to retain the existing FieldsIndexOffset and not introduce a separate sectionsIndexOffset. Do you think that's a good idea?

Reply:
If the user is going to remain oblivious of this stuff, then I think it's fine.

Reply:
i dont think it should affect the user in my opinion, since we'd not be advertising those two offsets (which are equal) separately and i dont think its necessary to expose it to user either actually.

Reply:
Ok, let's go with this for now, let's do our due diligence in testing this as well.